deploy/VeJoeStaking.js

```diff
@@ -0,0 +1,50 @@
+module.exports = async function ({ getNamedAccounts, deployments }) {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  let joeAddress,
+    veJoeAddress = (await deployments.get("VeJoeToken")).address,
+    baseGenerationRate = 0,
+    boostedGenerationRate = 1,
+    boostedThreshold = 1,
+    boostedDuration = 0,
```

Comment:
two small things on naming:
- we've renamed these from `boosted` -> `speedUp` in the contract, let's reflect that here.
- also rename `baseGenerationRate` -> `veJoePerSharePerSec` and `boostedGenerationRate` -> `speedUpVeJoePerSharePerSec`

Reply:
Thanks for this comment, I will rebase this once latest `VeJoeStaking` changes are merged and update variables.